plugins/MySQL.cpp

```diff
@@ -638,7 +652,7 @@ const char* g_MySQLVariables[MYSQL_NUM_VARIABLES][2] = {
     {"unique_checks", "ON"},
     {"updatable_views_with_limit", "YES"},
     {"version", "5.1.73-log"},
-    {"version_comment", SVERSION},
+    {"version_comment", "(Ubuntu)"},//SVERSION},
```

Comment:
maybe change the commented code into an english comment explaining why we are setting this as Ubuntu?